cpp/src/link_prediction/jaccard.cu

```diff
@@ -20,60 +20,67 @@
  * ---------------------------------------------------------------------------**/
 
 #include "utilities/graph_utils.cuh"
-#include "cugraph.h"
+#include "graph.hpp"
 #include "rmm_utils.h"
 #include "utilities/error_utils.h"
 
 namespace cugraph { 
 namespace detail {
+
   // Volume of neighboors (*weight_s)
-  template<bool weighted, typename IdxType, typename ValType>
+  template<bool weighted, typename vertex_t, typename edge_t, typename weight_t>
```

Comment:
Not a request to change, and just a comment.

Maybe a task for 0.14.

We're mixing VT, ET, WT and vertex_t, edge_t, and weight_t, and we'd better not do this.

We need to decide on template parameter names and should use those consistently.

Reply:
I introduced this to trigger conversation about the names.  Agree we should not mix things.  Hopefully we can address this at the graph alignment meeting.